rnd/autogpt_server/autogpt_server/data/execution.py

```diff
@@ -216,6 +215,18 @@ async def get_executions(graph_exec_id: str) -> list[ExecutionResult]:
     return res
 
 
+async def get_execution_result(
```

Comment:
This can be merged to `get_executions` by making node_exec_id optional?

Reply:
Yes it's been deleted on my local branch